src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -171,11 +171,15 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
                 const newProjectOption = options["new-project"]
                     ? "--new-project"
                     : "";
+
+                const shouldDebug = options.debug ?? false;
+                let debugFlag = "";
+                if (shouldDebug) debugFlag = "--debug";
                 const command = `dotnet --roll-forward Major ${resolvePath(
                     options.csharpGeneratorPath
                 )} --project-path ${outputFolder} ${newProjectOption} --clear-output-folder ${
                     options["clear-output-folder"]
-                }`;
+                } ${debugFlag}`;
```

Comment:
It will add extra space at the end of the command if shouldDebug==false.

Reply:
I moved the space into the value of variable `debugFlag`